scripts/stac/imagery/collection.py

```diff
@@ -0,0 +1,36 @@
+from typing import Any, Dict, List, Optional
+
+import ulid
+
+PYSTAC_VERSION = "1.0.0"
```

Comment:
Could be a good idea to move this constant to a common Python file that could be use by `item.py` too.